cpanfile

```diff
@@ -40,6 +40,7 @@ requires 'Term::ANSIColor';
 requires 'Test::Compile';
 requires 'Test::More';
 requires 'Test::Simple';
+requires 'Test::Warnings';
```

Comment:
The master already has this added into `on 'test'` section. Seems like you rebase is not up to date.